core/services/ardupilot_manager/ArduPilotManager.py

```diff
@@ -113,7 +113,7 @@ def start_mavlink_manager(self, device: Endpoint) -> None:
             self.add_new_endpoints(
                 {
                     Endpoint(
-                        "GCS Link", self.settings.app_name, EndpointType.UDPClient, "192.168.2.1", 14550, protected=True
+                        "GCS Link", self.settings.app_name, EndpointType.UDPClient, "192.168.2.2", 14550, protected=True
```

Comment:
this will break all current setups out there, right? we should talk about about it. Ideally we should be able to make both work. If we can't, this is the priority.

Reply:
Yes, altough current setups are all for testing :smile: 

I'm inclined to say it's a decision better to took now them later.

Reply:
let's keep the dhcp server running at 192.168.2.2, like companion 0